src/view.js

```diff
@@ -71,7 +71,9 @@ const View = Backbone.View.extend({
   // Subsequent renders after the first will re-render all nested
   // views.
   render() {
-    if (this._isDestroyed) { return this; }
+    const template = this.getTemplate();
+
+    if (this._isDestroyed || template === false) { return this; }
```

Comment:
as insignificant as this is, swapping these checks will be slightly more performant for the hotter route.